src/components/TabSelect/TabSelect.js

```diff
@@ -8,16 +8,23 @@ import TabButton from './_TabButton';
 
 const TabButtons = styled.div``;
 
-const TabContent = styled.div`
-  padding: 20px;
-  background-color: ${props => props.theme.colors.white};
-
-  ${props => `
+const borders = props => `
     border-top-right-radius: ${props.theme.borderRadius.soft};
     border-bottom-right-radius: ${props.theme.borderRadius.soft};
     border-bottom-left-radius: ${props.theme.borderRadius.soft};
     border: 1px solid ${props.theme.colors.athens};
-  `};
+`;
+
+const bordersOnlyTop = props => `
+  border-top: 1px solid ${props.theme.colors.athens}
+`;
+
+const TabContent = styled.div`
+  padding: 20px;
+  background-color: ${props =>
+    props.secondary ? props.theme.colors.sand : props.theme.colors.white};
```

Comment:
Also here - why not use `transparent` instead of `colors.sand`?

Reply:
same as above